src/gui/mzroll/analytics.cpp

```diff
@@ -81,6 +81,26 @@ void Analytics::httpPost(QUrlQuery query) {
 
 }
 
+QString Analytics::getClientID()
+{
+    QString settingsPath = QStandardPaths::writableLocation(QStandardPaths::ConfigLocation);
+    settingsPath = QDir::cleanPath(settingsPath + QDir::separator() + "El-MAVEN_analytics.ini");
+
+    QSettings settings(settingsPath, QSettings::IniFormat);
+    QString clientID;
+    if (!settings.contains("analytics-cid"))
+    {
+        clientID = QUuid::createUuid().toString();
+        settings.setValue("analytics-cid", clientID);
```

Comment:
@sahil21 Is the .ini file overwritten every time I install a new version? or does a user have a unique client ID across versions? Just curious.

Reply:
@shubhra-agrawal Unique id across versions.